tests/core/mempool/test_mempool_manager.py

```diff
@@ -92,6 +93,116 @@ async def instantiate_mempool_manager(
     return mempool_manager
 
 
+def make_test_conds(
+    *,
+    height_relative: Optional[uint32] = None,
+    height_absolute: uint32 = uint32(0),
+    seconds_relative: uint64 = uint64(0),
+    seconds_absolute: uint64 = uint64(0),
+) -> SpendBundleConditions:
+    return SpendBundleConditions(
+        [Spend(TEST_COIN.name(), IDENTITY_PUZZLE_HASH, height_relative, seconds_relative, [], [], 0)],
```

Comment:
I made `TEST_COIN_ID` for this one.

Reply:
I prefer to say `TEST_COIN.name()` because I want this to be the coin ID for `TEST_COIN`. Saying `TEST_COIN_ID` spends cognitive capacity by needing to find out what `TEST_COIN_ID` is, for no obvious gain.